apps/backend/src/datasources/postgres/ReviewDataSource.ts

```diff
@@ -372,7 +372,7 @@ export default class PostgresReviewDataSource implements ReviewDataSource {
               qb.where('instrument_has_proposals.instrument_id', instrumentId);
             }
 
-            if (status !== null) {
+            if (status !== null && status !== undefined) {

```

Comment:
```suggestion
            if (status) {
```

Reply:
Review status is of type:
`export enum ReviewStatus {
  DRAFT = 0,
  SUBMITTED = 1,
}` So could be 0 which would not pass this check when we need it too.